Dockerfile

```diff
@@ -6,11 +6,37 @@ FROM python:${PY_VERSION}-slim-bullseye AS compile-stage
 ARG VERSION
```

Comment:
I'm not sure if it's in scope but the `VERSION` build argument is not used in this image configuration.

Reply:
Yep I'm blind. Thanks for double-checking. I have a strong detestation for that argument.